### DISCUSSION TITLE
PCL存储库的第一个Issue是什么

### DISCUSSION THREAD

**Discussion by Ad-closeNN** [Meloong-Git/PCL#3396] [提问] [answered]
网页时光机也没有存档...
![waybackmachine](https://github.com/Hex-Dragon/PCL2/assets/103355026/d7ab6ba6-d8f7-4345-ae5e-fc9832ab43ad)

**Comment by WForst-Breeze** [comment, marked as answer]
测试机器人用的，见 #2906

**Comment by Ad-closeNN** [comment]
那第二个呢

**Reply by WForst-Breeze** [reply]
#1329 有回复

**Reply by iPodtouch6557** [reply]
~我甚至忘了自己发过这个（）~
